dotcom-rendering/src/components/Answers.tsx

```diff
@@ -14,6 +14,12 @@ import { palette as schemedPalette } from '../palette';
 // We export Radio wrapper styles to override Source Radio buttons to align
 // with our custom answers for the quiz
 export const radioButtonWrapperStyles = (theme: ArticleTheme) => css`
+	> fieldset > div > div {
+		padding: 0;
+		align-items: center;
+		margin-bottom: ${space[2]}px;
+	}
```

Comment:
The answer buttons would be better refactored to avoid needing to override so many radio button styles, but that feels like a separate task and out of scope for this update.